lib/image_optim/config.rb

```diff
@@ -33,7 +34,8 @@ def read_options(path)
         end
         return {} unless File.size?(full_path)
 
-        config = YAML.load_file(full_path)
+        args, kwargs = load_file_args(full_path)
+        config = YAML.load_file(*args, **kwargs)
```

Comment:
What do you think about extracting the method to load yaml file instead of a method to provide arguments for `YAML.load_file`?

In current way I completely understand comparing `YAML::VERSION` and `respond_to?` wouldn't be of help, but if there is a method instead that given path reads and returns yaml, it can check if there is a method to "safe load file" and use that, otherwise use `YAML.load_file`.